packages/odyssey-react-mui/src/Tabs.tsx

```diff
@@ -67,6 +67,17 @@ export type TabsProps = {
    * The TabItems to be included in the Tabs group
    */
   tabs: TabItemProps[];
+  /**
+   * Determine behavior of scroll buttons when tabs are set to scroll:
+   *
+   * - `auto` will only present them when not all the items are visible.
+   * - `true` will always present them.
+   * - `false` will never present them.
+   *
+   * By default the scroll buttons are hidden on mobile.
+   * @default 'auto'
+   */
+  scrollButtons?: MuiTabListProps["scrollButtons"];
```

Comment:
Boolean props should use `is` or `has` prefixes.

## Extra info
I couldn't tell what this was going through the whole PR until I came back to this point and read the comment.

The term "scroll buttons" is a command meaning it would probably be a function.

I also thought maybe it's passing components "these are the scroll buttons".

Reply:
renamed to `hasScrollButtons`

Reply:
and changed type to `auto|always|never`